### ISSUE TITLE
Editor Welcome Tour: Add new slide for List View

### ISSUE THREAD

**Issue by StefanNieuwenhuis** [Automattic/wp-calypso#56709]
**Note**: when picking this one, first check the changes from https://github.com/Automattic/wp-calypso/pull/56347 as mentioned in https://github.com/Automattic/wp-calypso/issues/56709#issuecomment-987785753
___

### Details
We'd like to introduce a new slide to the Editor Welcome Tour to present the List View feature.

https://user-images.githubusercontent.com/12104589/135470978-95e76aad-c565-4aff-89e4-910bbe40b000.mov

### Checklist (Acceptance Criteria)

- [x] Desktop (and mobile) copy @donalirl 
- [x] Desktop (and mobile) graphics @donalirl 
- [ ] Implementation - add this as the 6th slide, only on desktop resolutions
- [ ] Handle the i18n (hide it for non-EN and open a follow-up issue to display it when translations are ready)

**Comment by donalirl** [comment]
**Copy:**

_Title:_

Find your way

_Description:_

Use List View to see all the blocks you've added. Click and drag any block to move it around.

**GIF:**

![list view welcome tour](https://user-images.githubusercontent.com/30727666/136293341-5ebb320c-c7dd-4231-a059-fbb7bcafe14e.gif)

**Suggested position:**

It seems to make sense as the 6th slide, before 'Undo any mistake'

**Comment by donalirl** [comment]
The above is for desktop. From what I can tell, there is no List View on mobile unless I'm missing something?

**Comment by razvanpapadopol** [comment]
This should be unblocked now since the required [mobile/desktop logic has been introduced](https://github.com/Automattic/wp-calypso/pull/58658/commits/4b16993ba4668994e170e89303666dda2d12b645) by @escapemanuele in #58658

**Comment by StefanNieuwenhuis** [comment]
> This should be unblocked now since the required mobile/desktop logic has been introduced by @escapemanuele in #58658

We should wait for #56347 to be shipped before merging this one since it contains changes relevant to this issue.
